test/PolyhedralGeometry/subdivision_of_points.jl

```diff
@@ -14,9 +14,9 @@
     SubdivisionOfPoints
 
   @testset "alternative inputs" begin
-    @test issetequal(collect(maximal_cells(square_by_incidence)),
-      collect(maximal_cells(square_by_weights)))
-    @test min_weights(square_by_cells) == min_weights(square_by_weights)
+    @test issetequal(maximal_cells(square_by_incidence),
+                     maximal_cells(square_by_weights))
+    @test issetequal(min_weights(square_by_cells), min_weights(square_by_weights))
```

Comment:
```suggestion
    @test issetequal(square_max_cells, maximal_cells(subdivision_of_points(C,min_weights(square_by_cells))))
```
We should either remove this test or compare via maximal cells. But comparing the weights is still fragile.